docs/SG-NEx_m6Anet_tutorial.md

```diff
@@ -0,0 +1,151 @@
+
+To begin, we will create a directory to store the required files.
+```bash
+# create a directory to store the processed data for this sample
+mkdir A549_directRNA_replicate6_run1 
+```
+
+Then, we will access the S3 bucket to download the data into the created directory. There are two methods to download the data. 
+<br>
+
+#### **Method One(Recommended):** 
+Note: The command requires you to have [AWS CLI](https://aws.amazon.com/cli/) installed.
+```bash
+# To download the processed data
+aws s3 sync --no-sign-request s3://sg-nex-data/data/processed_data/m6Anet/SGNex_A549_directRNA_replicate6_run1 ./A549_directRNA_replicate6_run1
+# The directory should now contain three files. They are the json, index and readcount files.
+```
+Alternatively, we can use the URL to the S3 bucket to download the processed data. 
+<br>
+
+#### **Method Two:**
+```bash
+cd A549_directRNA_replicate6_run1 
+ 
+# To download the json file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/m6Anet/SGNex_A549_directRNA_replicate6_run1/data.json  
+
+# To download the index file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/m6Anet/SGNex_A549_directRNA_replicate6_run1/data.index
+
+# To download the readcount file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/m6Anet/SGNex_A549_directRNA_replicate6_run1/data.readcount
+
+# Exit the A549 directory to return to our working directory
+cd ..
+```
+
+
+You may refer to this [page](https://github.com/GoekeLab/sg-nex-data/blob/update-docs-aws/docs/samples_with_RNAmod_data.tsv) for the list of URLs to access the processed data for m6Anet from different samples. 
+
+## **Running m6Anet**
+
+Now that all the required files are ready, we can proceed to run the m6Anet analysis. 
+
+
+```bash
+# Run m6Anet
+m6anet-run_inference --input_dir ./A549_directRNA_replicate6_run1 --out_dir ./A549_directRNA_replicate6_run1 --infer_mod_rate --n_processes 4
+```
+**Note: You can reduce the "n_processes" if you have lesser processes available in your machine. However, lowering the "n_processes" may increase the run time.**
+
+
+m6Anet returns a csv file `(data.result.csv.gz)` that contains the probability of modification as well as the modification stoichiometry for each candidate site. The first few lines of the file are shown below.
+
+
+<p align="center">
+    <img src="../images/A549_directRNA_replicate6_run1_result.png" alt="result" width="800"/>
+</p>
+
+Please refer to the [m6Anet documentation](https://m6anet.readthedocs.io/en/latest/quickstart.html) for more information regarding the output file. 
+<br>
+
+
+m6Anet also allows the pooling of replicates. It can be done by providing the file path to each replicate's folder containing the json, index and readcount files. 
```

Comment:
```suggestion
m6Anet also allows the pooling of replicate samples. It can be done by providing the file path to each replicate sample's folder containing the json, index and readcount files. 
```